Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -177,7 +177,7 @@ public struct Glob {
           excludedSet.intersection(url.pathComponents).isEmpty
         else { continue }
 
-        directories.append(url)
+        directories.append(url.resolvingSymlinksInPath())
```

Comment:
The code change in `ApolloCodegen` to resolve symlinks does not actually solve the issue of `/private/var` vs `/var` paths in test because neither is marked as a symlink. However, running `.resolvingSymlinksInPath()` does remove `/private` from paths when it exists, so to be able to run tests for this issue I also had to change this line.

Thoughts on this? Or other suggestions of maybe handling this?